docs/quickstart.ipynb

```diff
@@ -172,23 +268,97 @@
        "4 -3.115669  0.790815  "
       ]
      },
-     "execution_count": 21,
+     "execution_count": 2,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
     "data = pf.get_data()\n",
+    "\n",
     "data.head()"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<class 'pandas.core.frame.DataFrame'>\n",
+      "RangeIndex: 1000 entries, 0 to 999\n",
+      "Data columns (total 11 columns):\n",
+      " #   Column    Non-Null Count  Dtype  \n",
+      "---  ------    --------------  -----  \n",
+      " 0   Y         999 non-null    float64\n",
+      " 1   Y2        1000 non-null   float64\n",
+      " 2   X1        999 non-null    float64\n",
+      " 3   X2        1000 non-null   float64\n",
+      " 4   f1        999 non-null    float64\n",
+      " 5   f2        1000 non-null   float64\n",
+      " 6   f3        1000 non-null   float64\n",
+      " 7   group_id  1000 non-null   float64\n",
+      " 8   Z1        999 non-null    float64\n",
+      " 9   Z2        1000 non-null   float64\n",
+      " 10  weights   1000 non-null   float64\n",
+      "dtypes: float64(11)\n",
+      "memory usage: 86.1 KB\n"
+     ]
+    }
+   ],
+   "source": [
+    "data.info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see that some of our columns have missing data."
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## OLS Estimation"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are interested in the relation between the dependent variable `Y` and the independent variables `X1` using a fixed effect model for `f1`. Let's see how the data looks like:"
```

Comment:
I was wondering if we should either replace `f1` here with `group_id`, as this seems to be more in the spirit of the explanation above if fixed effects as "parameters that are estimated to be constant across different groups"? 

Maybe we could even create a nicer synthetic dataset / find a nice proper example data set?

Reply:
I think this synthetic dataset is good enough for this intro section. I did change `f1` -> `group_id` (I was also going to suggest the change) in https://github.com/s3alfisc/pyfixest/pull/436/commits/eadcbf892160111010b44a6e8642ca220d690322

I actually wanna add another example: https://matheusfacure.github.io/python-causality-handbook/14-Panel-Data-and-Fixed-Effects.html This is how I learned about fixed effects in the first place and we have a public synthetic dataset with a panel data structure :) It is also nice to reproduce results from other references.

Reply:
Yes that's a really cool idea to replicate the main fixed effects example from "brave and true"! Agreed that the synthetic data set should do for the intro =)

Reply:
Added new example in https://github.com/s3alfisc/pyfixest/pull/436/commits/c12b44391e1dfbf25630440c56532c0fcce1f50f